src/popup/components/OverviewDetailsParty.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div
+    class="overview-details-party"
+    :data-cy="isRecipient ? 'recipient' : 'sender'"
+    :class="{ recipient: isRecipient }"
+  >
+    <a
+      v-if="txParty.url || !isRecipient"
+      :href="txParty.url"
+      target="_blank"
+      class="name"
+    >
+      <Truncate
+        :str="txParty.name || txParty.label"
+      />
+    </a>
+    <span
+      v-else
+      class="name"
+      :class="{ aens: txParty.aens }"
+    >
+      {{ txParty.label }}
+    </span>
+    <CopyText
+      v-if="txParty.address"
+      hide-icon
+      :value="txParty.address"
+    >
+      <span class="text-address">{{ splitAddress(txParty.address) }}</span>
+    </CopyText>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+import { splitAddress } from '../utils';
+import CopyText from './CopyText.vue';
+import Truncate from './Truncate.vue';
+
+export default defineComponent({
+  components: {
+    CopyText,
+    Truncate,
+  },
+  props: {
+    isRecipient: Boolean,
+    txParty: {
+      type: Object,
+      required: true,
+    },
+  },
+  setup() {
+    return {
+      splitAddress,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../styles/variables';
+@use '../../styles/typography';
+
+.overview-details-party {
+  width: 100%;
+
+  &.recipient {
+    .truncate {
+      justify-content: flex-end;
+    }
+  }
+
+  .name {
+    @extend %face-sans-15-medium;
+
+    display: block;
+    margin-bottom: 8px;
+    color: variables.$color-white;
+    text-decoration: none;
+    white-space: nowrap;
+    line-height: 16px;
+
+    &:hover {
+      .truncate::v-deep span {
```

Comment:
This line can be removed. The hover styling will be applied to the `name` class which is an anchor `<a>` so the underline will be applied.